### PR TITLE
Generate code and test for HealthBot

### PR DESCRIPTION
**Pull request by Youri970410**
## Description

Generate code and test for HealthBot

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/HealthBot/examples/Get-AzHealthBot.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+### Example 4: Get HealthBot by InputObject
+```powershell
+PS C:\> $getAzHealthBot = Get-AzHealthBot -ResourceGroupName youriTest -name yourihealthbot
+Get-AzHealthBot -InputObject $getAzHealthBot
+
+Location Name                 SystemDataCreatedAt SystemDataCreatedBy   SystemDataCreatedByType SystemDataLastModifiedAt SystemDataLastModifiedBy             SystemDataLastModifiedByType Type
+-------- ----                 ------------------- -------------------   ----------------------- ------------------------ ------------------------             ---------------------------- ----
+eastus   yourihealthbot       2020/12/29 5:54:14  test@microsoft.com User                    2020/12/29 5:54:19       ********-****-****-****-********** Application                  Microsoft.HealthBot/healthBots
+```
+```
```
Comment (isra-fel) [inline src/HealthBot/examples/Get-AzHealthBot.md:45]: Extra ```

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: src/HealthBot/examples/Get-AzHealthBot.md (hunk 1 above)
Comment (Youri970410) [reply src/HealthBot/examples/Get-AzHealthBot.md:45]: Got it.thanks for your reminder.

**Comment by isra-fel** [comment]
1. Please edit the base branch to `release-2021-01-19`
2. Please update this file as well: https://github.com/Azure/azure-powershell/blob/master/documentation/azure-powershell-modules.md

Otherwise LGTM.
 Thanks

**Comment by Youri970410** [comment]
@isra-fel Thanks,fixed.

**Review by isra-fel — APPROVED** [review]

**Pull request merged**